src/main/resources/public/ts/sniplets/content/workspace-nextcloud-toolbar-share.sniplet.ts

```diff
@@ -74,7 +74,10 @@ export class ToolbarShareSnipletViewModel implements IViewModel {
 
     async onCancelShareElements(): Promise<void> {
         if (this.sharedElement.length) {
-            await service.deleteAll(this.sharedElement);
+            service.deleteAll(this.sharedElement)
```

Comment:
qu'est-ce qui a changé ?
C'était quoi le pb ?

Reply:
Qaund ça fail avec le await on ne passait pas à la suite.

Reply:
try catch dans ce cas

```suggestion
  try {
     await service.deleteAll(this.sharedElement);
  } catch (e) {
  console.error((err));
  }
```
On sait quelles sont les types d'erreurs ?

Reply:
ok je fais ça